### PR TITLE
cloud: add managed SMTP documentation

### PR DESCRIPTION
**Pull request by bobheadxi**
Initial documentation for https://github.com/sourcegraph/customer/issues/1408.

Closes https://github.com/sourcegraph/customer/issues/1672, closes https://github.com/sourcegraph/customer/issues/1527

Preview: https://deploy-preview-5722--sourcegraph-handbook.netlify.app/departments/cloud/technical-docs/managed-smtp/

### CODE REVIEW

File: content/departments/cloud/technical-docs/managed-smtp/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- Alerting: [frontend: email_delivery_failures](https://docs.sourcegraph.com/admin/observability/alerts#frontend-email-delivery-failures)
+- Dashboards: [Frontend: Email delivery](https://docs.sourcegraph.com/admin/observability/dashboards#frontend-email-delivery)
+- [Multi-instance dashboard](../observability/index.md#multi-instance-dashboard): [Frontend: Total emails successfully delivered every 5 minutes](https://monitoring.sgdev.org/d/multi-instance-overviews/multi-instance-overviews?orgId=1)
+
+### Vendor-side
+
+Dashboards are available [in the SparkPost web application](https://app.eu.sparkpost.com/signals/analytics?range=day&timezone=America/Vancouver&precision=hour&metrics%5B0%5D=count_targeted&metrics%5B1%5D=count_rendered&metrics%5B2%5D=count_accepted&metrics%5B3%5D=count_bounce&report=summary). Relevant metrics to look at:
+
+- Accepted count: we are billed for this based on our usage plans, up to 250,000 (as of Dec 2022).
+- Bounce rates: this indicates emails that are being rejected by recipients due to misconfiguration (invalid recipients) or spam filters.
```
Comment (michaellzc) [inline content/departments/cloud/technical-docs/managed-smtp/index.md:42]: do we have alerting for increasing bounce rates?

Comment (bobheadxi) [reply content/departments/cloud/technical-docs/managed-smtp/index.md:42]: No, see below under "Future work" - in rollout `mi check` will periodically give bounce rate analytics (once a month as part of instance upgrade test plan). We will need a cron service to poll deliverability statistics, or a service to ingest webhook events, in order to do alerting

**Review by michaellzc — APPROVED** [review]

File: content/departments/cloud/technical-docs/managed-smtp/index.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+[Reporting access](https://support.sparkpost.com/docs/user-guide/managing-users) (also see [vendor management](#vendor-management)) via the web application and the [message events API](https://developers.sparkpost.com/api/events/#header-event-types) can be used to access:
+
+- Aggregate numbers (deliveries, bounces, etc)
+- Specific email events, including receipients, subjects, message sizes, and other diagnostic data (but **not** email contents)
+
+### Account isolation
+
+Our [vendor integration](#vendor-integration) is designed such that all usage and access can be constrolled on a per-subaccount (i.e. per-customer) basis. API tokens distributed to Cloud instances are scoped to individual subaccounts with very limited permissions, and can be disabled individually.
+
```
Comment (bobheadxi) [inline content/departments/cloud/technical-docs/managed-smtp/index.md:90]: cc @sourcegraph/security PTAL!

**Comment by bobheadxi** [comment]
Merging for now for a shareable reference page - access provisioning still WIP: https://github.com/sourcegraph/customer/issues/1660#issuecomment-1338380825 

Happy to address post-merge comments as well!

File: content/departments/cloud/technical-docs/managed-smtp/index.md (hunk 1 above)
Comment (bobheadxi) [reply content/departments/cloud/technical-docs/managed-smtp/index.md:42]: Follow-up: https://github.com/sourcegraph/customer/issues/1662

**Pull request merged**